Build Image Release Artefacts.yml

```diff
@@ -1,12 +1,22 @@
 name: Build Image Release Artefacts 3.0-$(Build.BuildId)
```

Comment:
Remove 3.0 from the Name property in the YAML.

Reply:
done, removed 3.0